scst-store/cluser-specific-scanner-configurations.hbs.md

```diff
@@ -0,0 +1,42 @@
+# Connecting Vulnerability scanning to SCST - Store 
+This page summarizes how to configure and connect your scanner with Supply Chain Security Tools - Store and provides links to more detailed explanations of individual topics.
+
+## Metadata Store
+ - Single cluster - Scanners will use `app-tls-cert` to communicate with SCST - Store
+ - Multicluster … Sacnners will use `ingress-cert` of SCST - Store in view cluster. Please refer [MultiCluster Setup](multicluster-setup.hbs.md)
+
+
+## Grype and other supported scanner configuration
```

Comment:
this also feels unnecessary.  what is the value add of this section?  it seems evident that configuration of the scanner is taken care of in the... configuration :)